README.md

```diff
@@ -218,6 +218,16 @@ $ fisher install blackjid/plugin-kubectl
 | kdpvc        | `kubectl describe pvc`                               |
 | kdelpvc      | `kubectl delete pvc`                                 |
 
+### Daemonset management.
+
+| Abbreviation | Command                                              |
+| ------------ | ---------------------------------------------------- |
+| kgds         | `kubectl get pvc`                                    |
```

Comment:
can you fix the readme so the command matches the real aliased command?

Reply:
Updated